src/compiler/tools/TypeChecker.ts

```diff
@@ -179,6 +180,27 @@ export class TypeChecker {
         return symbol == null ? undefined : this._context.compilerFactory.getSymbol(symbol);
     }
 
+    /**
+     * Gets the resolved signature from a node.
+     * @param node - Node to get the signature from.
+     * @returns undefined if signature cannot be resolved
+     */
+    getResolvedSignature(node: Node<ts.CallLikeExpression>): Signature | undefined {
+        const resolvedSignature = this.compilerObject.getResolvedSignature(node.compilerNode);
```

Comment:
I wonder what the other arguments are used for. It doesn't seem like a big deal to leave them out for now.

Reply:
I checked typescripts internal usage of this method but didn't make much sense of it, so I left them out